e2e-tests/tests.sh

```diff
@@ -33,7 +33,8 @@ trap finish EXIT
 
 # install the metadata-injection webhook
 helm repo add newrelic https://helm-charts.newrelic.com
-if ! helm upgrade --install "$HELM_RELEASE_NAME" newrelic/nri-metadata-injection \
+helm dependency update ../charts/nri-metadata-injection
```

Comment:
`update` creates the lock. `build` downloads the dependencies based on the lock.

Also it does not have dependencies (yet) and I saw that is already merged too 😅 

```suggestion
helm dependency build ../charts/nri-metadata-injection
```

Reply:
yes, but it will have the common library! I'll change it for build, thanks for pointing it out!